src/games/gameitem.py

```diff
@@ -218,99 +218,18 @@ def tooltip(self, data):
-
-    def _teams_tooltip(self, teams):
-        versus_string = (
-            "<td valign='middle' height='100%'>"
-            "<font color='black' size='+5'>VS</font>"
-            "</td>")
-
-        def alignment(teams):
-            for i, team in enumerate(teams):
-                if i == 0:
-                    yield 'left', team
-                elif i == len(teams) - 1:
-                    yield 'right', team
-                else:
-                    yield 'middle', team
-
-        team_tables = [self._team_table(team, align)
-                       for align, team in alignment(teams)]
-        return versus_string.join(team_tables)
-
-    def _team_table(self, team, align):
-        team_table_start = "<td><table>"
-        team_table_end = "</table></td>"
-        rows = [self._player_table_row(player, align) for player in team]
-        return team_table_start + "".join(rows) + team_table_end
-
-    def _player_table_row(self, player, align):
-        if isinstance(player, str):
-            country = "<td></td>"
-        else:
-            country = "<td>{country_icon}</td>"
-        pname = ("<td align='{alignment}' valign='middle' width='135'>"
-                 "{player}"
-                 "</td>")
-        order = [pname, country] if align == 'right' else [country, pname]
-        player_row = "<tr>{}{}</tr>".format(*order)
-
-        if isinstance(player, str):
-            return player_row.format(alignment=align, player=player)
-        else:
-            return player_row.format(
-                country_icon=self._country_icon_fmt(player),
-                alignment=align,
-                player=self._player_fmt(player))
-
-    def _country_icon_fmt(self, player):
-        icon_path_fmt = os.path.join("chat", "countries", "{}.png")
-        icon_path = icon_path_fmt.format(player.country.lower())
-        icon_abs_path = os.path.join(util.COMMON_DIR, icon_path)
-        return "<img src='{}'>".format(icon_abs_path)
-
-    def _player_fmt(self, player):
-        if player == self._me.player:
-            pformat = "<b><i>{}</b></i>"
-        else:
-            pformat = "{}"
-        player_string = pformat.format(player.login)
-        if player.rating_deviation < 200:   # FIXME: magic number
-            player_string += " ({})".format(player.rating_estimate())
-        return player_string
-
-    def _observers_tooltip(self, observers):
-        if not observers:
-            return ""
-
-        observer_fmt = "{country_icon} {observer}"
-
-        observer_strings = [observer_fmt.format(
-            country_icon=self._country_icon_fmt(observer),
-            observer=observer.login)
-            for observer in observers]
-        return "Observers: " + ", ".join(observer_strings)
-
-    def _mods_tooltip(self, mods):
-        if not mods:
-            return ""
-        return "<br/>With: " + "<br/>".join(mods.values())
+        template_abs_path = os.path.join(util.COMMON_DIR, "games", "gameitem.css")
```

Comment:
This is something that the theme class should provide us (opening and giving us file contents). I think it's better to use it. Also .css file ending is very unfortunate, we should change it so something like .jinja or .qhtml or the like.

Reply:
yes, i was wondering whats up with the .css too - i just took the naming convention from the other jinja template (chat) :D